CONTRIBUTING.md

```diff
@@ -123,6 +123,47 @@ post = Post.create(image: Down.download("https://example.com/image-from-internet
 # Your code for reproducing
 ```
 
+Appendix C: Running the Templates
+=================================
+ 
+* Someone has submitted a bug report: how then do we debug and run it?
+
+You have two options: 
+
+* Using a Gemfile
+
+* Inlining the required gems.
+
+### Using a Gemfile
+
+```ruby
+# Gemfile
+gem "rails"
+gem 'shrine'
+gem "down"
+```
+
+* Run `bundle install` and then you can simply run the template
+* `ruby template_name.rb`
+
+### Inlining the required gems:
+
+```ruby
+## Append the following to the top of the ruby template script
+require 'bundler/inline'
+
+gemfile do
+  source 'https://rubygems.org'
+  gem "rails"
+  gem 'shrine'
+  gem "image_processing"
+  gem "sqlite3"
+end
+```
+
+* Now run `ruby template_name.rb` and the script should run.
+
```

Comment:
Lastly let's add a third section:

----
### Debugging Shrine code

If you would like to debug or step through Shrine's code using the above template scripts, follow these steps:
* Download (or clone) Shrine's repository to your local machine
`git clone ...`
* Checkout the version of shrine you are using by running `git checkout <version_tag>`
* Modify the `Shrine` gem line in your `Gemfile` to point to the local copy of Shrine code
`gem "shrine", path: "/path/to/your/local/shrine/code"`
* If needed, add a debugger like `byebug` or `pry` to the `Gemfile`
* Run `bundle install`
* Add `require "bundler/setup"` to the top of the template script
* You can add debugger statement anywhere in your template file or Shrine code
* Run `ruby template_name.rb`.

Reply:
thx got it working with the 'bundler/setup' line.